### DISCUSSION TITLE
[contributing] [submod] new songs

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#9607] [general - submod / external content]
hi! submitting some song ideas for the next release, which i coded as a submod~

list of songs:

- 1, 2, 3, 4, by plain white t's;
- 2 step, by ed sheeran;
- a whole new world, from aladdin;
- better than heaven, by jhana boy;
- call me irresponsible, by bobby darin;
- deadly dangerous love, from yandere simulator;
- digital girl, by kira;
- digital love, by daft punk;
- everything i do (i do it for you), by bryan adams;
- hey stephen, by taylor swift;
- i'm not in love, by 10cc;
- i think i kinda, you know, by olivia rodrigo;
- last night on earth, by green day;
- let's get married, by mitski;
- love is easy, by mcfly;
- love like you, by rebecca sugar;
- love me like you, by little mix;
- make you mine, by public;
- me and my husband, by mitski;
- nobody, by mitski;
- no one has to know, by shy baldwin;
- photograph, by ed sheeran;
- someone you like, by the girl and the dreamcatcher;
- sunflower, by rex orange county;
- sweet tooth, by cavetown;
- tee shirt, by birdie;
- tek it, by cafune;
- this side of paradise, by coyote theory;
- uwu, by chevy;
- you are the reason, by calum scott.

[Otter's Songs.zip](https://github.com/Monika-After-Story/MonikaModDev/files/9142690/Otter.s.Songs.zip)

**Comment by ThePotatoGuy** [comment]
oops we'll turn this back into an issue when we need to do it. 

waiting on #9608
